composer.json

```diff
@@ -64,14 +64,17 @@
         "knplabs/knp-gaufrette-bundle": "~0.1",
         "symfony-cmf/routing-bundle": "~2.0",
         "fpn/doctrine-extensions-taggable": "~0.9",
+        "twig/twig": "^1.12|^2.0",
```

Comment:
This line should be removed, correct `twig/twig` dependency is set on line 68/69